pytorch_lightning/core/saving.py

```diff
@@ -136,39 +136,43 @@ def load_from_checkpoint(
         return model
 
     @classmethod
-    def _load_model_state(cls, checkpoint: Dict[str, Any], *cls_args, **cls_kwargs):
+    def _load_model_state(cls, checkpoint: Dict[str, Any], *cls_args_new, **cls_kwargs_new):
         cls_spec = inspect.getfullargspec(cls.__init__)
         cls_init_args_name = inspect.signature(cls).parameters.keys()
         # pass in the values we saved automatically
         if cls.CHECKPOINT_HYPER_PARAMS_KEY in checkpoint:
-            model_args = {}
+            cls_kwargs_old = {}
```

Comment:
consider rename as the opls may invoke the old hparams keys 
```suggestion
            cls_kwargs_loaded = {}
```